Sources/FirebaseFirestore/Query+Swift.swift

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+@_exported
+import firebase
+@_spi(Internal)
+import FirebaseCore
+
+import CxxShim
+
+public typealias Query = firebase.firestore.Query
+
+extension Query {
+  var firestore: Firestore {
+    swift_firebase.swift_cxx_shims.firebase.firestore.query_firestore(self)
+  }
+
+  func getDocuments(completion: @escaping (QuerySnapshot?, Error?) -> Void) {
+    let future = swift_firebase.swift_cxx_shims.firebase.firestore.query_get(self, .default)
```

Comment:
Does the `.default` need to get bubbled up into a parameter of `getDocuments` signature with the default value of `.default`?

Reply:
Yeah, that's a good idea. I was just implementing the variant without that parameter.

Reply:
I'll defer this change to the follow-up PR that fleshes out more of the Query API.